docs/battle_anim_commands.md

```diff
@@ -84,7 +84,7 @@ Since there can't be two of the same bg effect, the effect type is used. This is
 
 ## `$DA`: `anim_battlergfx_1row`
 
-Loads animation graphics for the bottom one or two rows of the enemy pokemon and the top one or two rows of the player's.  
+Loads animation graphics for the bottom one or two rows of the enemy pokemon and the top one or two rows of the player's.
```

Comment:
Two trailing spaces in Markdown act as a line break.

Reply:
Ah wasn't aware. I'll fix that.